Assets/Project/Scripts/Utils/AddressableAssetManager.cs

```diff
@@ -238,6 +225,20 @@ internal static void LoadStageDependencies(ETreeId treeId, int stageNumber)
                     }
                 });
             });
+
+            stage.GimmickDatas.ForEach(gimmick => {
+                switch (gimmick.type) {
+                    case EGimmickType.Tornado:
+                    case EGimmickType.RandomTornado:
+                        LoadAsset<GameObject>(Address.TORNADO_PREFAB);
+                        LoadAsset<Sprite>(Address.TURN_CARTRIDGE_WARNING_SPRITE);
```

Comment:
ちゃんと Rename すれば，ここが 
```suggestion
                        LoadAsset<Sprite>(Address.TURN_TORNADO_WARNING_SPRITE);
```
になるはず？